### PR TITLE
Add Opus codec support 

### PR DESCRIPTION
**Pull request by smolskyaleksey**

### CODE REVIEW

File: Pod/vialer-pjsip-include/pjmedia-codec/config_auto.h [hunk 1]
```diff
@@ -87,7 +87,7 @@
 
 /* OPUS codec */
 #ifndef PJMEDIA_HAS_OPUS_CODEC
-#define PJMEDIA_HAS_OPUS_CODEC 0
+#define PJMEDIA_HAS_OPUS_CODEC 1
```
Comment (bobvoorneveld) [inline Pod/vialer-pjsip-include/pjmedia-codec/config_auto.h]: I don't think this is needed when you've added this flag already in config_site.h. Please try without and let me know if you can do without.

File: vialerbuild [hunk 2]
```diff
@@ -28,18 +28,21 @@ vialerbuild info arm64    # show architecture information for arm64 libs.
 vialerbuild info i386     # show architecture information for i386 libs.
 vialerbuild info x86_64   # show architecture information for x86_64 libs.
 
-vialerbuild complete	  # Downloads, builds and copies files.
-vialerbuild rebuild		  # Delete old framework files and rebuild for all architectures
+vialerbuild complete ABSOLUTE_PATH_TO_DEPENDENCY_LIB   # Downloads, builds and copies files with opus lib
+vialerbuild rebuild	ABSOLUTE_PATH_TO_DEPENDENCY_LIB	  # Delete old framework files and rebuild for all architectures with opus lib
```
Comment (bobvoorneveld) [inline vialerbuild]: The positioning of the arguments become more important at this moment, and it is required now to add a path to an opus directory, which might not be needed by everyone who is using the script. I think we need a better solution for these kind of options.

**Comment by bobvoorneveld** [comment]
Hi @smolskyaleksey, I see you're still pushing commit to this PR. Can you give us a sign as soon as you want us to review again?

**Comment by smolskyaleksey** [comment]
Pls, close this one. I will create a new one, when all notes will fixed.

**Pull request closed without merging**